Cargo.toml

```diff
@@ -34,5 +34,11 @@ log = "0.4.22"
 env_logger = "0.11.5"
 thiserror = "1.0.63"
 moka = { version = "0.12.8", optional = true, features = ["future"] }
-strum = { version = "0.26.3", features = ["derive"] }
+# strum = { version = "0.26.3", features = ["derive"] }
```

Comment:
There a reason we leaving this here commented out? Just wondering since its being replaced with `derive_more`